### PR TITLE
One click integration

### PR DESCRIPTION
**Pull request by mdkalish**
Make everything compatible with new permissions structure

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -5,7 +5,8 @@ gem 'rails', '~> 4.2'
 gem 'attr_extras'
 gem 'awesome_print'
 gem 'celluloid'
-gem 'data_guru'
+gem 'data_guru', git: 'https://github.com/netguru/data_guru',
```
Comment (siepet) [inline Gemfile]: I think it's better to have modifications to Gemfile and Gemfile.lock in separate commit.

File: app/models/toggl_integration/team_repository.rb [hunk 2]
```diff
@@ -12,7 +12,7 @@ def self.build_from_data_guru(dg_client, user_repository, toggl_members_reposito
         tasks = team.tasks.try(:map) { |task| Task.new(id: nil, name: task, pid: team.id) }
         Team.new(name: team.name,
                  members: members || [],
-                 projects: team.projects,
+                 projects: team.project,
```
Comment (siepet) [inline app/models/toggl_integration/team_repository.rb]: `projects: team.project` - team doesn't have more than one project?

**Comment by siepet** [comment]
:shipit:

File: app/models/toggl_integration/team_repository.rb (hunk 2 above)
Comment (mdkalish) [inline app/models/toggl_integration/team_repository.rb]: Yeah, we talked it over with @lubieniebieski and decided that having multiple projects is a zero-use case.

Comment (siepet) [inline app/models/toggl_integration/team_repository.rb]: I see! :+1:

Comment (lubieniebieski) [inline app/models/toggl_integration/team_repository.rb]: we will rename it in next iteration, don't worry

**Pull request merged**